js/script.js

```diff
@@ -1186,11 +1186,18 @@ Array.prototype.unique = function() {
 		favArray : [],
 		add : function(){
 			var latlng = $(this).attr("data-latlng").split(",");
-			var formData = {
-				lat : latlng[0],
-				lng : latlng[1]
-			};
-			$.post(OC.generateUrl('/apps/maps/api/1.0/favorite/addToFavorites'), formData);
+			OC.dialogs.prompt("Enter a name", "Name your favorite", function(isYes, name){
```

Comment:
Should the input field appear in the popup after the user clicked on "Add to favorites"?